spec/index.bs

```diff
@@ -8846,12 +8846,11 @@ The <dfn dfn>encoder state</dfn> may be one of the following:
 
 <div algorithm>
     To <dfn abstract-op>Enqueue a command</dfn> on {{GPUCommandsMixin}} |encoder|
-    which issues the steps of a [=GPU Command=] |command|, optionally with [=RenderState=] |renderState|:
```

Comment:
After your changes no instance of `Enqueue a command` make use of the `renderState`, so I've removed it. (It was only used previously in the `setBindGroup` algorithm.